packages/terra-table/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-table",
+  "main": "lib/index.js",
+  "version": "4.36.0",
+  "description": "The terra-table component provides user a way to render selectable data in a tabular format",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/table-framework.git",
+    "directory": "packages/terra-table"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-table",
+    "Table",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/table-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/table-framework#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": "2 || 3 || 4 || 5"
+  },
+  "dependencies": {
+    "@cerner/terra-docs": "^1.9.0",
+    "classnames": "^2.2.5",
+    "focus-trap-react": "^6.0.0",
+    "keycode-js": "^3.1.0",
+    "prop-types": "^15.5.8",
+    "resize-observer-polyfill": "^1.4.1",
+    "terra-content-container": "^3.39.1",
+    "terra-icon": "^3.54.0",
+    "terra-mixins": "^1.40.0",
```

Comment:
Are we using terra-mixins in this project?  If not, it should not be a dependency.